src/main/java/com/synopsys/integration/alert/channel/hipchat/HipChatChannel.java

```diff
@@ -224,14 +224,12 @@ private String createHtmlMessage(final String contentTitle, final String content
             } else {
                 templateDirectoryPath = System.getProperties().getProperty("user.dir") + "/src/main/resources/hipchat/templates";
             }
-
             final ChannelFreemarkerTemplatingService freemarkerTemplatingService = new ChannelFreemarkerTemplatingService(templateDirectoryPath);
 
             final HashMap<String, Object> model = new HashMap<>();
-            model.put("content", content);
-            model.put("contentTitle", contentTitle);
+            model.put("content", messageContent);
 
-            return freemarkerTemplatingService.getResolvedTemplate(model, "audit.ftl");
```

Comment:
Should I delete the old notification.ftl?

Reply:
Obviously not the audit one.

Reply:
yes you can delete the old notification.ftl since it is no longer used.  The audit.ftl file was originally to display the raw content.  That may not be needed anymore either.